src/reducers/index.js

```diff
@@ -38,7 +38,8 @@ export const populatePropertyDefaults = (propertyTemplate) => {
   if (propertyTemplate === undefined || propertyTemplate === null || Object.keys(propertyTemplate).length < 1) {
     return defaults
   }
-  if (propertyTemplate.valueConstraint.defaults && propertyTemplate.valueConstraint.defaults.length > 0) {
+
+  if (propertyTemplate?.valueConstraint?.defaults && propertyTemplate.valueConstraint.defaults.length > 0) {
```

Comment:
Because you're using the safe navigation operator here, you can drop the first condition and shorten this to:

```javascript
if (propertyTemplate?.valueConstraint?.defaults?.length > 0) {
```

Safe navigation will return `null` anywhere along the chain that an attribute isn't defined, and `null` is not greater than 0, so the condition does what we want.